runtime/j9vm/jvm.c

```diff
@@ -2233,9 +2233,13 @@ JNI_GetCreatedJavaVMs(JavaVM **vmBuf, jsize bufLen, jsize *nVMs)
  */
 
 jint JNICALL JNI_GetDefaultJavaVMInitArgs(void *vm_args) {
-	UDATA requestedVersion = (UDATA)((JDK1_1InitArgs *)vm_args)->version;
+	JDK1_1InitArgs *init_args = (JDK1_1InitArgs *)vm_args;
+	UDATA requestedVersion = (UDATA)init_args->version;
```

Comment:
`JDK1_1InitArgs` should only be used _after_ determining that the caller specified `JNI_VERSION_1_1`.
Please remove the cast to `UDATA`; using `jint` is preferable.

Reply:
Changed it to cast to `JavaVMInitArgs` by default, and only using `JDK1_1InitArgs` after requested version is `JNI_VERSION_1_1`